static/app/views/replays/list.tsx

```diff
@@ -35,10 +49,22 @@ function ReplaysListContainer() {
         <Layout.Body>
           <Layout.Main fullWidth>
             <LayoutGap>
-              <ReplaysFilters />
-              <ReplaysErroneousDeadRageCards />
-              <ReplaysSearch />
-              <ReplaysList />
+              <ReplayListPageHeaderHook />
+              {fetching ? null : showOnboarding ? (
+                <Fragment>
+                  <ReplaysFilters>
+                    <ReplaysSearch />
+                  </ReplaysFilters>
+                  <ReplayOnboardingPanel />
```

Comment:
Intentional to not show the deadrage cards in this case?

Reply:
Yup.

Today in the app, when the onboarding panel appears, you're going to see these 4 things all stacked up like blocks:
```
<Filters>
<DeadRageCards>
<Search>
<Onboarding>
```

It's weird though to show the DeadRageCards, because they either won't have any content, and take up a lot of space, or they'll have leftover content from when the trial was active. 

Either way, they're taking up space and it pushes the onboarding thing down below the fold.

So in this PR I ripped the cards out, and made the filters+search look good again (on one row) when onboarding is shown. After onboarding is done, you'll see the stack of filters + cards + search + table.